Rakefile

```diff
@@ -136,6 +137,35 @@ nugets_pack :create_nuget_quick => [:versioning, 'build/pkg'] do |p|
   end
 end
 
+task :paket => [:versioning, 'build/pkg'] do
+   puts "starting"
+   files = FileList['src/**/*.fsproj'].exclude(/Tests/)
+   files.each do |f|
+    p = Albacore::Project.new f
+    m = "type file
+id #{p.id}
+version #{ENV['NUGET_VERSION']}
+authors Ademar Gonzalez, Henrik Feldt
+description #{suave_description}
+language en-GB
+copyright Ademar Gonzalez, Henrik Feldt
+licenseUrl https://github.com/SuaveIO/Suave/blob/master/COPYING
+projectUrl https://raw.githubusercontent.com/SuaveIO/resources/master/images/head_trans.png
+iconUrl https://raw.githubusercontent.com/SuaveIO/resources/master/images/head_trans.png
+files
+    #{p.proj_path_base}/bin/#{Configuration} ==> lib"
```

Comment:
Nice, does it work for CoreCLR? I don't mind if it doesn't right now as we're not publishing nugets from such a compilation.